compose/src/main/java/com/bottlerocketstudios/compose/snippets/SnippetsMockData.kt

```diff
@@ -0,0 +1,11 @@
+package com.bottlerocketstudios.compose.snippets
+
+import com.bottlerocketstudios.compose.util.StringIdHelper
+
+internal val snippet1 = SnippetUiModel(
```

Comment:
@VanyaHuamanBottleRocket Refactor this to something like `mockSnippet1` or similar